### ISSUE TITLE
prometheus input plugin - 'url' label/tag gets overwritten with the scrape url. Need configurable tag name for scrape url

### ISSUE THREAD

**Issue by vishiy** [influxdata/telegraf#5798]
## Feature Request
 'url' label/tag gets overwritten with the scrape url. need configurable tag name for scrape url
Opening a feature request kicks off a discussion.

### Proposal:
Introduce a config setting 'scrapeUrlTagName' (if not specified it will be 'url' as it is today)
### Current behavior:
always use 'url' as tagname for scrapeurl
### Desired behavior:
make the tag name configurable
### Use case: 
some of the k8s API Server & kubelet metrics all have 'url' as labels
ex;-
`# TYPE rest_client_request_latency_seconds histogram
rest_client_request_latency_seconds_bucket{url="https://127.0.0.1:443/%7Bprefix%7D",verb="GET",le="0.001"} 230605`

**Comment by danielnelson** [comment]
:+1: Let's call the option `url_tag`.

**Comment by vishiy** [comment]
@danielnelson - I have a fix for this (small change as you may know). Should i pile into the existing prometheus metrics v2 PR 5767 ? Is that cool or wait until its merged to put this on top of that ?

**Comment by danielnelson** [comment]
Either way would be fine.

**Comment by andheiberg-gelato** [comment]
Any updates on this issue? Just hit it now for the first time :(

**Comment by vishiy** [comment]
@danielnelson is working on merging this. Not sure about the latest ETA.

**Comment by danielnelson** [comment]
Closed in #5767

**Comment by i-prudnikov** [comment]
Hello guys, the configuration settings for this feature
```
  ## Url tag name (tag containing scrapped url. optional, default is "url")
  # url_tag = "scrapeUrl"
 ```
 is missed from readme.md.
